nuv/main.go

```diff
@@ -33,6 +33,7 @@ type CLI struct {
 	Kind       KindCmd       `cmd:"" help:"kind subcommand"`
 	Devcluster DevClusterCmd `cmd:"" help:"create or destroy kind k8s cluster"`
 	Setup      SetupCmd      `cmd:"" passthrough:"" help:"setup nuvolaris"`
+	Version    VersionCmd    `cmd:"" help:"version"`
```

Comment:
Hey, @Deviad i would prefer to have _nuv --version_ instead of _nuv version_
Could you add _passthrough:" "_ option here?